plugins/chat/lib/chat/steps_inspector.rb

```diff
@@ -80,6 +80,9 @@ def error
 
     # @!visibility private
     class Policy < Step
+      def error
+        step_result.reason
```

Comment:
With this small change, the steps inspector is able to show the reason why a policy failed.